content/events/2018-kansas-city/welcome.md

```diff
@@ -4,11 +4,34 @@ Type = "welcome"
 aliases = ["/events/2018-kansas-city"]
 Description = "devopsdays Kansas City 2018"
```

Comment:
Most places we use DevOpsDays as opposed to devopsdays.

Reply:
I used their tooling to generate this automatically. I vaguely remember them taking about this, but don't really remember it enough. They wanted to go all lowercase on the site. I'm happy to change it. It looks weird to me.